src/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenCommonParameters.cs

```diff
@@ -25,18 +25,16 @@
 // 
 // ------------------------------------------------------------------------------
 
-namespace Microsoft.Identity.Client.ApiConfig
+using System.Collections.Generic;
+using Microsoft.Identity.Client.TelemetryCore;
+
+namespace Microsoft.Identity.Client.ApiConfig.Parameters
 {
-    /// <summary>
-    /// </summary>
-    internal interface IAcquireTokenOnBehalfOfParameters : IAcquireTokenCommonParameters
+    internal class AcquireTokenCommonParameters
     {
-        /// <summary>
-        /// </summary>
-        UserAssertion UserAssertion { get; }
-
-        /// <summary>
-        /// </summary>
-        bool SendX5C { get; }
+        public ApiEvent.ApiIds ApiId { get; set; } = ApiEvent.ApiIds.None;
+        public IEnumerable<string> Scopes { get; set; }
+        public Dictionary<string, string> ExtraQueryParameters { get; set; }
+        public string AuthorityOverride { get; set; }
     }
```

Comment:
seems like LogParameters is not available here. should it be?

Reply:
Good point, and I thought about this.  But CommonParameters currently get turned into AuthenticationRequestParameters which does the logging for these.  So for now, it would be redundant.  

---
In reply to: [249086194](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/798#discussion_r249086194) [](ancestors = 249086194)